File: Week1/mario-more/mario.c
#include<stdio.h>
#include<cs50.h>

void space(int n);
void brick(int n);

int main()
{
    //prompt user for input
    int height;
    do{
        height=get_int("Height: ");
    }while(height<1 || height>8);

    //print pyramids
    for(int i=0;i<height;i++)
    {
        //spaces
        space(height-i-1);

        //bricks
        brick(i+1);
        printf("  ");
        brick(i+1);
        printf("\n");

    }
}

void space(int n)
{
    for(int i=0;i<n;i++)
    {
        printf(" ");
    }
}
void brick(int n)
{
    for(int i=0;i<n;i++)
    {
        printf("#");
    }
}

File: Week6/sentimental-mario-less/mario.py
while True:
    try:
        height = int(input("Height: "))
        if 0 < height < 9:
            break
        print("Invalid Input")
    except ValueError:
        print("Invalid Input")

for i in range(height):
    for space in range(height-i-1):
        print(" ", end="")

    for brick in range(i+1):
        print("#", end="")

    print()
